components/builder-notify/src/hook.rs

```diff
@@ -0,0 +1,32 @@
+use crate::client::WebhookClient;
```

Comment:
I can definitely see the desire to abstract `hook` but until we have more than one implementation, it may be clearer to keep it simple and do away with the `hook` trait until we need it.